tests/tests-inconsistency/pom.xml

```diff
@@ -78,6 +78,17 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-failsafe-plugin</artifactId>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <executions>
+          <execution>
+            <goals>
+              <goal>test-jar</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
```

Comment:
Why do you need the test-jar?

Reply:
I wanted to subclass the classes used for evaluation that this model provides. However, since the classes are in src/test, they are not part of the build and cannot be used as a dependency without telling maven to create a test-jar. Classes in the test subdirectory are supposed to test classes in the main subdirectory. The tlr and id test modules violate these principles. In retrospect, fixing this problem by moving the main code into the main subdirectory seems like a more elegant and sensible solution. After that, the snippet above can be removed